tests/CMakeLists.txt

```diff
@@ -88,6 +102,13 @@ if(PROGRESS_GRAPHLIB)
   progress_test(prg_build_zmatGP)
 endif()
 
+# c test modules
+progress_test(prg_density_c)
```

Comment:
All the c test names end with "_c" in order to avoid conflicts with Fortran test names.